### ISSUE TITLE
Remove the code change added to the officer search

### ISSUE THREAD

**Issue by sbhamani** [PublicDataWorks/complaint-manager#328]
### Business Context

The development made when this card was played is no longer necessary. Therefore, we need to search the codebase and remove the changes that were added.

In scope: 
- [x] Remove environment variable OFFICER_ROSTER_LATEST_DATE from `config.yaml`
- [x] Remove related tests
- [x] Remove implementation from `searchOfficers.js`
- [x] Remove any other implementation related to this work. 
- [x] Update [ADR](https://sites.google.com/thoughtworks.com/tw-invisible-institute-wiki/adrs) to include this outcome. (view how to document proposals that we decided to not continue the implementation)

## The context below was kept for contextualization
UPDATE: THIS FEATURE IS NO LONGER NEEDED. NEED REVERT ALL CHANGES ADDED TO A CODEBASE 

### Business Context ###
After a looong analysis, we learned that our current officers table in prod has a ton of duplicated data. In order to avoid displaying the same officers when someone does a search for officers, we decided to upload the most recent roster and flag their created_at date as defined date and only show those created at defined date in the UI when someone creates a new case. 

### In Scope ###
1. Include the list of officers attached on this ticket. **Make sure the created_at column reflect defined date.**  Data for this table is in the csv file attached.
2. Change the UI to show only officers that has the latest created_at (defined date) to be selected when someone creates a new case.
3. Check for differences from PROD and Non-PROD environments.
4. Update CSV to prod ENV for NOIPM.

Note: 
* When someone is looking at existing cases they should be able to see whatever officer's date was used at that time, meaning we only want to restrict showing officers that are from the latest created_at date when they are creating a new case, and not making that restriction for any other situation (search, looking at old cases, export, audit logs, etc).
* Consider the idea of creating a new table which holds the date that will be the starting date in the search of officers when creating a new case. 

Note 2: 
**This may be happen again in the future when we receive new officers data**

[Officers CSV  - Sheet1.csv](https://github.com/PublicDataWorks/complaint-manager/files/13746563/Officers.CSV.-.Sheet1.csv)

- [x] Find Search function to display newest officer names 
- [x] we want if statement to check only for env=prod for updated at time and only show new officers for creating new cases 
- [x] Look into searchOfficers.js 
- [x] Add error handling 
- [x] Write a test to check if function is working correctly 

### Additional Acceptance Criteria ###
- [x] Create an ADR in the Wiki and develop standards for ADRs going forwards

### QA Notes ###
- Make sure that when searching in the main search bar, duplicated officers still appear

**Comment by mariliasoaresTW** [comment]
We are updating the officers table today. If officers were added before and were used on cases we can't just remove them from the system.

**Comment by rnair-tw** [comment]
Notes from kickoff
- This is a one time change
- Need a future story to allow client to update the officer inmates themselves
- When get get the new spreadsheet (potentially some time in May?), should get into call w/ New Orleans to go through our DB
- Shak/Reshmi to do some deep dive into the workflow to understand how to improve process moving fwd

**Comment by heliofreitastw** [comment]
The story #458 was created to make the import of the data in the database

**Comment by heliofreitastw** [comment]
This architecture decision record was created with the solution implemented: https://sites.google.com/thoughtworks.com/tw-invisible-institute-wiki/adrs

**Comment by rnair-tw** [comment]
blocked on the foreign key issue
